### PR TITLE
zephyr: add k64f

### PR DESCRIPTION
**Pull request by EmbeddedAndroid**
Adds a run for the k64f

Signed-off-by: Tyler Baker <tyler@foundries.io>

### CODE REVIEW

File: zephyr.yml [hunk 1]
```diff
@@ -18,6 +18,18 @@ triggers:
...
           path: zephyr/sanity.sh
+      - name: sanity-k64f
+        container: zephyrprojectrtos/zephyr-build
+        host-tag: zephyr-k64f
+        privileged: true
+        params:
+          ZEPHYR_TOOLCHAIN_VARIANT: zephyr
+          PLATFORM: frdm_k64f
+          BROKE_TESTS: |
+            tests/kernel/interrupt
```
Comment (doanac) [inline zephyr.yml:30]: Do you need to add " samples/basic" here so that they won't hang/fail?

**Comment by EmbeddedAndroid** [comment]
Yeah that is probably a good idea for now. I'm looking how to fix this upstream but we can carry this for now. I assume the blenano2 isn't running the sample/basic/* tests. Will update.

**Comment by EmbeddedAndroid** [comment]
I added samples/basic, but it still runs the tests. Debugging now.

**Comment by EmbeddedAndroid** [comment]
heh, had to update the simulate script, as the broke test were hard coded in the rundef. Works now!

**Pull request merged**